de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/docs/generics/report_footer.jrxml

```diff
@@ -109,11 +110,11 @@ ad_org.ad_org_id = $P{org_id} AND ad_org.isActive = 'Y'
 				<textElement textAlignment="Center">
 					<font size="7"/>
 				</textElement>
-				<textFieldExpression><![CDATA[//$F{org_bank_name} + " | " +$R{routingno} + $F{org_bank_blz} + " | " + $R{swift} + $F{org_bank_swift}
+				<textFieldExpression><![CDATA[//$F{org_bank_name} + " | " + $R{swift} + $F{org_bank_swift} + " | " + $R{iban} + $F{org_bank_iban}
```

Comment:
@metas-lc just a sidenote 
we had trouble with groovy in the past, but i you want to try it out again:, there is the [elvis-operator](http://docs.groovy-lang.org/latest/html/documentation/#_elvis_operator) in groovy that might make such code easier